packages/formatter-codeframe/src/formatter.ts

```diff
@@ -11,20 +11,20 @@
  * ------------------------------------------------------------------------------
  */
 
-import chalk from 'chalk';
+import chalk, { Chalk } from 'chalk';
```

Comment:
Is `chalk` still needed now that it's in `utils`?

Reply:
yes, we still use `chalk.cyan` to format the resource.